### PR TITLE
Review PET Templates.

### PR DESCRIPTION
**Pull request by salvoscala**

### CODE REVIEW

**Comment by salvoscala** [comment]
@acrolloet The goal of this task is to update PET templates and give to guests and owners all the informations they need via mail. I've added some tokens and text to templates.

Emails are not recreated with a Drupal update. The site owner will receive an email with a link to update pets and he's informed that this action will delete all templates in the site (including translated ones) and recreate them with new layouts.

File: modules/roomify/roomify_accommodation_booking/roomify_accommodation_booking.tokens.inc [hunk 1]
```diff
@@ -32,6 +40,75 @@ function roomify_accommodation_booking_tokens($type, $tokens, array $data = arra
...
+          $image = '';
+          $name = '';
+          $short_description = '';
+          $name = $property->name;
+          if ($featured_image = field_get_items('roomify_property', $property, 'field_sp_featured_image')) {
+            $image = array('style_name' => 'square', 'path' => $featured_image[0]['uri']);
+            $image['attributes'] = array('style' => 'width:270px;border: 2px solid white;vertical-align:bottom;');
+            $short_description = field_get_items('roomify_property', $property, 'field_sp_short_description');
+          }
+          $return[$original] =
```
Comment (acrollet) [inline modules/roomify/roomify_accommodation_booking/roomify_accommodation_booking.tokens.inc:99]: would it make sense to use a template file here?

File: modules/roomify/roomify_system/roomify_system.install [hunk 2]
```diff
@@ -826,3 +826,30 @@ function roomify_system_update_7051() {
...
+/**
+ * Notify about new templates.
+ */
+function roomify_system_update_7053() {
+  $url = url('admin/bat/pet/update', array('absolute' => TRUE));
+  $site_email = variable_get('site_mail', '');
+  $site_name = variable_get('site_name', '');
+
+  $message = 'Hello ' . $site_name . '!
+    we have made some changes to email templates that are sent through your site. We did not udpdate them cause we have to delete all existing templates (including translation) and recreate them with a new layout and text.</br>
```
Comment (acrollet) [inline modules/roomify/roomify_system/roomify_system.install]: cause -> because

File: modules/roomify/roomify_system/roomify_system.install [hunk 3]
```diff
@@ -826,3 +826,30 @@ function roomify_system_update_7051() {
...
+    If you want to update them visit this ' . l(t('page'), $url) . '
```
Comment (acrollet) [inline modules/roomify/roomify_system/roomify_system.install]: If you wish to update them, please visit this

File: modules/roomify/roomify_system/roomify_system.module [hunk 4]
```diff
@@ -230,6 +239,54 @@ function roomify_footer_form($form, &$form_state) {
...
 /**
+ * Form "Update Pets".
+ */
+function roomify_update_pets_form($form, &$form_state) {
+  $updated = variable_get('roomify_pets_updated', FALSE);
+
+  if (!$updated) {
+    $form['intro'] = array(
+      '#prefix' => '<div style="width:100%;font-size:18px;line-height:1.3em;margin-bottom:30px;">',
+      '#markup' => t('Hi! We have made some changes to email templates. Clicking here you will delete your templates and import the new ones. Be carefult that this action will also delete your translated templates!'),
```
Comment (acrollet) [inline modules/roomify/roomify_system/roomify_system.module]: "Please be aware that this action will delete your translated templates!"

**Review by acrollet — CHANGES_REQUESTED** [review]
CR passes pending this review

**Pull request merged**